routers/api/packages/nuget/nuget.go

```diff
@@ -111,13 +112,14 @@ func getSearchTerm(ctx *context.Context) string {
 
 // https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedQueryBuilder.cs
 func SearchServiceV2(ctx *context.Context) {
-	skip, take := ctx.FormInt("skip"), ctx.FormInt("take")
+	skip, take := ctx.FormInt("$skip"), ctx.FormInt("$top")
```

Comment:
`$skip`  what is the significance of `$` in the word.
`$top` is it `top` or `stop`. could you annotate this in comment.

Reply:
That's just how the spec defines this parameter. The names are `$skip` and `$top`. Sadly one older client uses `skip` and `take` and that's the reason why they are mentioned as fallback here. I moved the official names to the top.

Reply:
may be small comment in the code base will be help ful.